### PR TITLE
Issue #67 - Snippet Rotation + Fix null snippet bug

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This is only rotation non-reward snippets for now, and also fixes the issue caused in #245 where snippet content isn't loaded.

Issue ref: #67

### CODE REVIEW

File: data/js/about-home.js [hunk 1]
```diff
@@ -36,12 +36,7 @@ function showSnippet(snippetContent) {
 self.port.on('modify', function(snippetContent) {
     // see whether a default snippet container exists
     if (defaultSnippetContainer) {
-        // try to grab the stuff we're going to insert into the page
```
Comment (chrismore) [inline data/js/about-home.js]: @FrancescoSTL @schalkneethling we need to test this to ensure this isn't going to duplicate content inside of the funnelcake.

**Comment by chrismore** [comment]
I've tested this out locally and it works as expected.

**Comment by chrismore** [comment]
Also resolves issue #68

File: randomsnippetcopy [hunk 2]
```diff
@@ -0,0 +1,40 @@
+function randomSnippetCopy(track, contentURL) {
```
Comment (schalkneethling) [inline randomsnippetcopy]: Not sure of the purpose of this file?

File: index.js [hunk 3]
```diff
@@ -643,6 +700,35 @@ function toggleSidebar() {
...
+    var snippetContent;
+    var copyNumber = Math.floor(Math.random() * 3) + 1;
+
+    console.log(copyNumber);
+
+    switch(copyNumber) {
+        case 1:
+            // load snippet HTML
+            snippetContent = self.data.load(contentURL).replace('%url', self.data.url(imageURL)).replace('%snippetCopy', SNIPPET_COPY[track][sidebarProps.step - 1].copy1);
+            console.log(snippetContent);
```
Comment (schalkneethling) [inline index.js]: Remove console.log

File: index.js [hunk 4]
```diff
@@ -643,6 +700,35 @@ function toggleSidebar() {
     }
 }
 
+function replaceSnippetCopy(track, contentURL, imageURL) {
+    var snippetContent;
+    var copyNumber = Math.floor(Math.random() * 3) + 1;
+
+    console.log(copyNumber);
```
Comment (schalkneethling) [inline index.js]: Remove console.log

**Comment by schalkneethling** [comment]
Couple of nits, but looking good r+ w/c

**Comment by FrancescoSTL** [comment]
Fixed
